vllm/model_executor/models/minicpmv.py

```diff
@@ -909,9 +960,9 @@ def get_vision_embedding(
         )
         return vision_embedding
 
-    def get_vision_hidden_states(
-            self, data: MiniCPMVImagePixelInputs) -> torch.Tensor:
-        pixel_values = data["pixel_values"]
+    def get_vision_hidden_states(self,
+                                 data: MiniCPMVImageInputs) -> torch.Tensor:
+        pixel_values = data["data"]
```

Comment:
```suggestion
        pixel_values = data["pixel_values"]
```

Is this a typo?

Reply:
No, the data field is a unified format of XXImageEmbeddingInputs for multimodal models. e.g. InternVLImageEmbeddingInputs.

Reply:
Then shouldn't the functions above also use `data` key?

Reply:
Goctha!